docs/prometheus-exporter.md

```diff
@@ -123,3 +123,22 @@ Possible values of `result` are:
 - `"invalid"` Messages being rejected as invalid.
 - `"dropped"` Messages being dropped due to a full queue. See either `network_inbound_low_priority_message_queue_size` or `network_inbound_high_priority_message_queue_size` for more on this.
 - `"duplicate"` Duplicate consensus messages (this is not including deduplication of the network layer).
+
+### `network_soft_banned_peers`
+
+Current number of soft banned peers. The node temporarily ban peers if they fail to follow the protocol.
+
+### `network_peers_total`
+
+Total number of peers connected since startup. This is incremented when a peer completes a handshake.
```

Comment:
"completes a handshake" is the definition of becoming a peer as we use it in the network layer.

Before that it is just a "connection". So the second part of the sentence does not add value. 

We could additionally have a metric (maybe we do already) on "connections received or established". That would be useful information.

Reply:
So I have removed the last sentence.

We have the metric `network_connections_received_total` which is counting every "attempt" at connecting, this counts even if the connection is banned, duplicate or the node is at its limit.
If you mean something different from this, then we should add it to the metric wishlist.